profiles/common/modules/features/cce_basic_config/cce_basic_config.module

```diff
@@ -325,3 +325,32 @@ function cce_basic_config_tmgmt_dgt_html_filter_formats_containing_html() {
     'full_html',
   );
 }
+
+/**
+ * Implements hook_preprocess_container().
+ *
+ * It preprocesses the container theme in order to inject
+ * theme_hook_suggestions.
+ */
+function cce_basic_config_preprocess_container(&$variables) {
+  $element = $variables['element'];
+  $suggestion = 'container';
+
```

Comment:
wysiwyg is not the bundle but the display.

The bundle in our case is : video, audio, document and image